packages/components/image-viewer/src/image-viewer.vue

```diff
@@ -37,6 +37,7 @@
               </el-icon>
             </span>
           </template>
+          <span :class="[ns.e('btn'), ns.e('progress')]">{{ progress }}</span>
```

Comment:
It can support slots, making it more flexible to use.
```suggestion
       
          <span :class="[ns.e('btn'), ns.e('progress')]">
            <slot name="progress" :active-index="activeIndex" :total="urrlList.length">
              {{ progress }}
            </slot>
          </span>
```